docs/03-proof_submission_interface/03-polygon_cdk_proof_submission.md

```diff
@@ -64,4 +64,4 @@ In order to do this you can use [PolkadotJs](https://polkadot.js.org/apps/?rpc=w
 
 ## Step 4
 
-Check your transaction on the [ZkVerify Block Explorer](https://nh-explorer.horizenlabs.io/v0/extrinsic)!
\ No newline at end of file
+Check your transaction on the [ZkVerify Block Explorer](https://nh-explorer.horizenlabs.io/v0/extrinsic)!
```

Comment:
Just to be sure that we didn't miss this. `ZKVerify Block Explorer` or `Proof Verification Chain Block Explorer`?

Reply:
zkVerify is correct.